nav_msgs/OccupancyGrid map
-----------------
geometry_msgs/Point[] points
int8 k

